classes/migration/upgrade/PKPv3_3_0UpgradeMigration.inc.php

```diff
@@ -595,13 +595,13 @@ private function _createCustomBlockTitles() {
 				->where($contextDao->primaryKeyColumn, $row->context_id)
 				->first()
 				->primary_locale;
-			$blocks = json_decode($row->setting_value);
+			$blocks = unserialize($row->setting_value);
 			foreach ($blocks as $block) {
 				$newRows[] = [
 					'plugin_name' => $block,
 					'context_id' => $row->context_id,
 					'setting_name' => 'blockTitle',
-					'setting_value' => json_encode([$locale => $block]),
+					'setting_value' => serialize([$locale => $block]),
```

Comment:
@asmecher should we be converting these instances of serialized db data into json or keep them as serialized?